plover/gui_qt/add_translation.py

```diff
@@ -153,12 +151,61 @@ def _translation(self):
         translation = self.translation.text().strip()
         return unescape_translation(translation)
 
+    def _update_items(self, dictionaries=None, reverse_order=None):
+        if dictionaries is not None:
+            self._dictionaries = dictionaries
+        if reverse_order is not None:
+            self._reverse_order = reverse_order
+        iterable = self._dictionaries
+        if self._reverse_order:
+            iterable = reversed(iterable)
+        self.dictionary.clear()
+        for d in iterable:
+            item = shorten_path(d.get_path())
+            if not d.enabled:
+                item += ' [' + _('disabled') + ']'
```

Comment:
That's because I messed up in the engine configuration update code when rewriting, so dictionaries are not actually disabled...

Note that the add translation dialog does not watch for configuration changes for the `dictionaries` variable, so the enabled/disabled state of dictionaries is not updated...